test/src/test/groovy/org/openremote/test/protocol/http/HttpClientProtocolTest.groovy

```diff
@@ -613,11 +616,10 @@ class HttpClientProtocolTest extends Specification implements ManagerContainerTr
         String[] dynamicTimeTests = ["\${time:PT1H:yyyy-MM-dd HH:mm}", "\${time:PT2H:yyyy-MM-dd HH:mm:ss}", "\${time:P1D:yyyy-MM-dd}", "\${time:P2DT3H:yyyy-MM-dd HH:mm}", "\${time:PT30M:yyyy-MM-dd HH:mm}", "\${time:P7D:yyyy-MM-dd}", "\${time:PT5H:yyyy-MM-dd HH:mm}", "\${time:P2D:yyyy-MM-dd}", "\${time:PT1M:yyyy-MM-dd}", "\${time:P3D:yyyy-MM-dd}"] as List<String>
 
         then: "The strings are parsed correctly. If there is a way to use a constant timestamp, it'd be much easier."
```

Comment:
Not sure what is meant about a way to use a constant timestamp would be much easier

Reply:
Being able to somehow set a static time that is returned, so that I can generate the literal strings that should be returned instead of having to write code to adapt it to current time.